### PR TITLE
Patryk/dev 5 add types to btn component

### PR DESCRIPTION
**Pull request by Afanasi0101**

### CODE REVIEW

File: quiz.code-workspace [hunk 1]
```diff
@@ -3,6 +3,9 @@
     {
       "name": "🌱 root",
       "path": "."
+    },
+    {
+      "path": "../Nowy folder"
```
Comment (aleksanderwalczuk) [inline quiz.code-workspace:8]: remove it. How many times do I have to write it? Please don't waste my time.

File: src/components/btn.js [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
   const universalBtn = document.createElement('button');
   universalBtn.textContent = text;
-  universalBtn.classList.add('first_btn');
+  universalBtn.className = btnTypes[btnType];
+
   return universalBtn;
 }
-
-export default createBtn;
+export default (createBtn);
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:16]: remove parenthesis

File: src/scss/components/btn.scss [hunk 3]
```diff
@@ -1,16 +1,41 @@
-.first_btn{
-  padding: 1rem 5rem;
-  border-radius: 9px;
-  font-size: 1.5rem;
+.btn_all{
```
Comment (aleksanderwalczuk) [inline src/scss/components/btn.scss:1]: set this name to `btn`

File: src/scss/components/btn.scss [hunk 4]
```diff
@@ -1,16 +1,41 @@
...
   color: $color-white;
   box-shadow: 0 0 15px $color-red;
   border: 1px solid $color-red;
   text-transform: uppercase;
-  cursor: pointer;
+}
+.primary_btn{
+  width: 600px;
+}
+.large_primary_btn{
```
Comment (aleksanderwalczuk) [inline src/scss/components/btn.scss:21]: `primary_btn_large` not `large_primary_btn`

File: src/components/btn.js [hunk 5]
```diff
@@ -1,8 +1,16 @@
-function createBtn(text) {
+const btnTypes = {
+  primary: 'primary_btns btn_all primary_btn',
+  primaryLarge: 'primary_btns btn_all large_primary_btn',
+  base: 'btn_all base_btn',
+  smallBase: 'btn_all small_base_btn',
+  baseLarge: 'btn_all large_base_btn',
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:6]: adjust classes names according to previous comments

File: src/components/btn.js [hunk 6]
```diff
@@ -1,8 +1,16 @@
...
+};
+
+function createBtn(text, btnType) {
   const universalBtn = document.createElement('button');
   universalBtn.textContent = text;
-  universalBtn.classList.add('first_btn');
+  universalBtn.className = btnTypes[btnType];
+
   return universalBtn;
 }
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:15]: We should check if provided `btnType` exists in our `btnTypes` object. If not then we should set the btnType as `base`.

**Review by aleksanderwalczuk — CHANGES_REQUESTED** [review]
This looks promising, but there are still some things we have to do:
- error prevention, check here https://github.com/bright-bounty/quiz/pull/10/files#r974607198
- we're missing the navigation button type
- there are few minor naming mistakes in css
- changes in workspace file should disappear ASAP

File: quiz.code-workspace [hunk 7]
```diff
@@ -3,7 +3,7 @@
     {
       "name": "🌱 root",
       "path": "."
-    }
+    },
```
Comment (aleksanderwalczuk) [inline quiz.code-workspace:6]: ...

File: src/components/btn.js [hunk 8]
```diff
@@ -1,8 +1,17 @@
-function createBtn(text) {
+const btnTypes = {
+  primary: 'primary_btns btn primary_btn',
+  primaryLarge: 'primary_btns btn primary_btn_large',
+  base: 'btn base_btn',
+  smallBase: 'btn base_btn_small',
+  baseLarge: 'btn base_btn_large',
+  navBtn: 'nav_btn',
```
Comment (aleksanderwalczuk) [inline src/components/btn.js:7]: it's all about buttons, so no need to add `Btn` to type name. Should be just `nav`

File: src/scss/components/btn.scss [hunk 9]
```diff
@@ -1,16 +1,51 @@
...
+  height: 100px;
+  border-radius: 16px;
+  font-size: 42px;
+  line-height: 51.2px;
+  text-align: center;
+  font-weight: 500;
+  cursor: pointer;
+  transition: color .5s, background-color 1s;
+}
+.primary_btns{
```
Comment (aleksanderwalczuk) [inline src/scss/components/btn.scss:11]: this class name should be singular not plural

Comment (aleksanderwalczuk) [reply src/scss/components/btn.scss:11]: we should keep it as following:
```css
.primary_btn {
  background-color: $color-red;
  color: $color-white;
  box-shadow: 0 0 15px $color-red;
  border: 1px solid $color-red;
  text-transform: uppercase;
  cursor: pointer;
  width: 600px;
}

File: src/scss/components/btn.scss [hunk 10]
```diff
@@ -1,16 +1,51 @@
...
   background-color: $color-red;
   color: $color-white;
   box-shadow: 0 0 15px $color-red;
   border: 1px solid $color-red;
   text-transform: uppercase;
-  cursor: pointer;
+}
+.primary_btn{
+  width: 600px;
+}
```
Comment (aleksanderwalczuk) [inline src/scss/components/btn.scss:20]: this should be removed. We can keep `width: 600px` and override `width` using `.primary_btn_large`

File: src/scss/components/btn.scss [hunk 11]
```diff
@@ -1,16 +1,50 @@
...
   text-transform: uppercase;
   cursor: pointer;
+}
+.primary_btn_large{
+  width: 701px;
+  height: 75px;
 
 }
-.first_btn:hover{
+.primary_btn:hover, .primary_btn_large:hover{
```
[the remark this answers is not in the document]
Comment (Afanasi0101) [reply src/scss/components/btn.scss:25]: Why not ? it's a separate button

**Review by aleksanderwalczuk — APPROVED** [review]
🚀

**Pull request merged**